webcompat/static/css/src/issue-wizard.css

```diff
@@ -1041,6 +1047,10 @@
     border-color: var(--wizard-step-error);
   }
 
+  .issue-form .text-field:focus {
+    box-shadow: 0 0 0 3px #56638c;
+  }
+
```

Comment:
Since this rule located under `@media (min-width: 665px)`, it will currently apply only to desktop, so it should be moved up (similarly to where `.issue-form .is-validated .text-field` is ).  

Also probably better to use `var(--issue-type-bg)` as a box-shadow color